Rakefile

```diff
@@ -31,4 +33,8 @@ end
 require 'rubocop/rake_task'
 RuboCop::RakeTask.new
 
-task default: [:rubocop, :spec]
+if RUBY_VERSION >= '2.6'
```

Comment:
Why? Just to save Running rubocop? This feels like it assumes content in .travis.yml and will be hard to debug. Furthermore anyone contributing on Ruby 2.5.3 will forget that Rubocop runs until they push and get a failure. How about changing the `rake` parameters in `.travis.yml` and running `rubocop` for only one matrix line in there?

Reply:
Alright, I was able to add a separate build that only runs `rake rubocop` and the default `rake` now only runs the `spec` task.